### ISSUE TITLE
The fields parameter doesn't appear in API docs

### ISSUE THREAD

**Issue by daviddavis** [wimglenn/djangorestframework-queryfields#25]
The `fields` and `fields!` parameters don't appear in DRF's autogenerated api docs. It'd be nice to have code and/or docs on how to add these parameters in djangorestframework-queryfields.

**Comment by zomglings** [comment]
@wimglenn : Are you accepting PRs on this repo? I plan to work on this since I'm using your library and also need to have an OpenAPI schema that correctly reflects the query parameter.

Without having done too much research, I assume I can replicate django-filter's strategy: https://django-filter.readthedocs.io/en/master/guide/rest_framework.html#schema-generation-with-core-api-and-open-api

Let me know.

**Comment by wimglenn** [comment]
Yes, I am accepting PRs.  The auto-generated docs stuff seems like a useful addition, just make sure to add test coverage for any new code.
